lesson9/task1.js

```diff
@@ -0,0 +1,29 @@
+//task 1
+const character = { name: "Barney", age: 36, gender: "male", isQa: true };
+const wordsWithFourSimbols = [
+  Object.keys(character).filter((el) => el.length == 4),
+];
```

Comment:
Не нужно дополнительно обворачивать в массив
- Object.keys возвращает массив
- filter возвращает массив
```suggestion
const wordsWithFourSimbols = 
  Object.keys(character).filter((el) => el.length == 4);
```